app/src/main/java/com/lagradost/cloudstream3/utils/UIHelper.kt

```diff
@@ -123,6 +129,20 @@ object UIHelper {
         )
     }
 
+    fun clipboardHelper(label: UiText, text: CharSequence) {
+        try {
+            val clip = ClipData.newPlainText(label.asString(context!!), text)
+            val labelSuffix = txt(R.string.toast_copied)
+            context?.getSystemService<ClipboardManager>()!!.setPrimaryClip(clip)
+
+            if (Build.VERSION.SDK_INT <= Build.VERSION_CODES.S_V2) {
+            showToast("${label.asString(context!!)} ${labelSuffix.asString(context!!)}")
+            }
+        } catch (t: Throwable) {
+            Log.e("ClipboardService", "$t")
+            showToast(R.string.clipboard_too_large)
```

Comment:
This is not true, it can crash on multiple things like context!! or getSystemService!!

Reply:
better handling now